src/uu/join/src/join.rs

```diff
@@ -60,13 +60,110 @@ enum FileNum {
     File2,
 }
 
-#[derive(Copy, Clone, PartialEq)]
-enum Sep {
-    Char(u8),
+#[derive(Clone)]
+enum SepSetting {
+    Byte(u8),
+    Char(Vec<u8>),
```

Comment:
please document these two, the diff isn't obvious :)

Reply:
Sure, done.